drivers/i3c/i3c_common.c

```diff
@@ -1350,6 +1350,24 @@ int i3c_bus_init(const struct device *dev, const struct i3c_dev_list *dev_list)
 	bool need_daa = true;
 	bool need_aasa = true;
 	struct i3c_ccc_events i3c_events;
+	struct i3c_config_controller ctrl_cfg;
+
+	/* Retrieve the active controller configuration */
+	ret = i3c_config_get(dev, I3C_CONFIG_CONTROLLER, &ctrl_cfg);
+	if (ret != 0) {
+		LOG_ERR("Failed to retrieve controller configuration");
+		return ret;
+	}
+
+	/* Set OD timings for Slow speed */
+	ctrl_cfg.scl_od.high_ns = I3C_OD_FIRST_BC_THIGH_MIN_NS;
+	ctrl_cfg.scl_od.low_ns  = I3C_OD_TLOW_MIN_NS;
```

Comment:
this should be left as is, no need to write the value here for the `high_ns`.... only the `high_ns`

but it probably should be checking if the `scl_od_min.high_ns` already meets the timing
```suggestion
	ctrl_cfg.scl_od.low_ns  = MAX(I3C_OD_TLOW_MIN_NS, ctrl_cfg.scl_od.low_ns);
```

Reply:
Can you elaborate about high_ns setting here

Reply:
Sorry, I meant to write "no need to write the value here for the low_ns"